config/setupTests.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+
+global.React = React;
+global.window = Object.create(window);
+global.window.insights = {
+  ...(window.insights || {}),
+  chrome: {
+    auth: {
+      getUser: () =>
+        new Promise((res) =>
+          res({
+            identity: {
+              // eslint-disable-next-line camelcase
+              account_number: '0',
+              type: 'User',
+            },
+            entitlements: {
+              insights: {
+                // eslint-disable-next-line camelcase
+                is_entitled: true,
+              },
+            },
+          })
+        ),
+    },
+    isBeta: jest.fn(),
+    getUserPermissions: () =>
+      new Promise((res) =>
+        res([
+          {
+            permission: 'drift:*:*',
```

Comment:
I know it is just a test set up, but this probably should not be drift here? Maybe for now just: `() => Promise.resolve()
    },`

Reply:
good catch